src/test/java/reposense/git/GitLogTest.java

```diff
@@ -94,8 +94,8 @@ public void gitLog_sinceDateInFuture_noContent() {
         Assert.assertTrue(content.isEmpty());
 
         date = TestUtil.getDate(1950, Calendar.JANUARY, 1);
-        config.setUntilDate(date);
-        config.setSinceDate(null);
+        config.setSinceDate(date);
+        config.setUntilDate(null);
```

Comment:
Is this change part of the PR?

Reply:
We want to keep this PR with only the required changes to make repo location an optional field.

Also, it is quicker for us to verify and merge the fix, if it is in a separate PR, so other contributors can pull. :-)